lua/ttt2/extensions/player.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- player extension
```

Comment:
Actually, i dislike these functions, as they all do not really belong to the player library in my opinion, as they rather look like functions, with game logic, which is more suited in the appropriate code of the game state

Reply:
I though `player` does stuff to all players. Like `player.GetAll()`